inlong-manager/manager-dao/src/main/resources/mappers/InlongGroupEntityMapper.xml

```diff
@@ -211,9 +211,8 @@
     <select id="selectDeletedGroupIds" resultType="java.lang.String">
         select inlong_group_id
         from inlong_group
-        where modify_time &lt;= #{lastModifyTime,jdbcType=TIMESTAMP}
-        group by inlong_group_id
-        having min(is_deleted) > 0
+        where is_deleted > 0
```

Comment:
It could be possible that there are two groups of the same inlong_group_id and one of them is deleted but the other one is not, in which case the non-deleted one will also be selected and incorrectly cleansed.

Reply:
Oh, I didn't think this case. I will add it back.